spec/models/user_spec.rb

```diff
@@ -12,4 +12,13 @@
   describe 'associations' do
     it { should have_many(:targets) }
   end
+
+  describe '#chats' do
+    it 'should return chats collections' do
+      user_a = create(:user)
```

Comment:
I would suggest creating this user outside the `it` block, and inside the `describe` block.